docs/HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md

```diff
@@ -171,6 +181,5 @@ private transactions to create a contract.
 
 <!-- links ---->
 
-[privacy marker transaction]: ../../Concepts/Privacy/Private-Transaction-Processing.md
```

Comment:
Given that you use this link 3 times in the file, having it here only once may be nice.

Reply:
Yes, cutting down on the number of links in the file.